rocky/reports/views/base.py

```diff
@@ -119,6 +120,7 @@ def get_report_types(self) -> List[ReportType]:
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         context["observed_at"] = self.valid_time
+        context["created_at"] = datetime.now()
```

Comment:
You might want to either convert this to a UTC timestamp, or whenever rendering this string also mention the timezone. Maybe this is the intended change, but it's not clear from the submitted screenshot